test/src/e2e_vm_tests/test_programs/should_pass/language/test_multiple_attributes/src/main.sw

```diff
@@ -2,4 +2,3 @@ library test_multiple_attributes;
 
 #[test, inline(always)]
 fn foo() {}
```

Comment:
maybe we should add some more complicated formatter tests that involve more attributes or duplicate attributes?

Reply:
Added a bit better unit test for the formatter and refactored the simple one in `swayfmt/src/formatter/mod.rs`. We don't deal with duplicate attributes at the moment, unfortunately -- I opened an issue on that https://github.com/FuelLabs/sway/issues/3598

Reply:
Also, once we have the `payability` attribute, I can add more formatter tests (there is not too many attributes to make complex examples right now)